Fix: Added Issue and Pull Request Template

VividhPandey003 opened the pull request:
Closes: #181 

Created an issue template for the repository to encourage easy contribution.
Created two readme files for the issue template i.e bug_report and feature_request.

Rajdip019 commented:
The issue template looks awesome. Bur please make the PR to dev branch not main. @VividhPandey003

VividhPandey003 commented:
> The issue template looks awesome. Bur please make the PR to dev branch not main. @VividhPandey003

Hey @Rajdip019 Sorry for that. I have changed the base branch to dev. I have also added a pull request template. Could you please review this pull request now along with the new changes I have made

Rajdip019 commented:
This looks good. @VividhPandey003 .

Congrats 🎉, your PR got merged. To be eligible for the swags from iNeuron X ClueLess. Follow this steps.

1. SignIn / SignUp in [ClueLess Website](https://www.clueless.tech/).
2. Go to profile section from the top right hand corner.
    
    ![https://s3.us-west-2.amazonaws.com/secure.notion-static.com/74339abf-adaf-4b25-8496-7642f08f7c03/Screenshot_2022-09-30_at_2.40.58_PM.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20220930%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220930T091323Z&X-Amz-Expires=86400&X-Amz-Signature=346a58118a3686075c16e368b225e3c9ec7cb48c7a90c70903ff788e0812832f&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22Screenshot%25202022-09-30%2520at%25202.40.58%2520PM.png%22&x-id=GetObject](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/74339abf-adaf-4b25-8496-7642f08f7c03/Screenshot_2022-09-30_at_2.40.58_PM.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20220930%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220930T091323Z&X-Amz-Expires=86400&X-Amz-Signature=346a58118a3686075c16e368b225e3c9ec7cb48c7a90c70903ff788e0812832f&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22Screenshot%25202022-09-30%2520at%25202.40.58%2520PM.png%22&x-id=GetObject)
    
3. Comment your profile URL under this comment tagging the person who merged your PR.

That’s it. Follow the leaderboard !

And don’t forget to share your achievement with your friends.
